### ISSUE TITLE
CPKT File and Dataset 

### ISSUE THREAD

**Issue by yashrajeshhiray** [manumathewthomas/CS523Project3#12]
Hello sir, please share CPKT file and Dataset

**Comment by cenkburakegeli** [comment]
Yes please. The links that you provided are not working. Can you share them from google drive? Thanks for the code and the help

**Comment by yp19940913** [comment]
Did you find  CPKT file and Dataset? Can you share it? thank you very much!!!!!!!!!! @yashrajeshhiray @manumathewthomas @cenkburakegeli

**Comment by kushjayankpandya** [comment]
I think the link for CKPT file is broken? Can you share the drive link?
